modules/loadbalancer/outputs.tf

```diff
@@ -20,6 +20,7 @@ output "frontend_combined_rules" {
   ```
   {
     "frontend01-balancessh" = {
+      backend_port = null
```

Comment:
why do we explicitly set this to `null`

Reply:
This is the output example. However in output_rules in locals we set backend_port to null if it's not provided. That's because the NSG rules are created with a for_each over output_rules in which we specify the destination port to match with the backend port if provided, or to match the front end port if not provided. Related line:

https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/blob/239-support-for-frontend-and-backend-ports-in-loadbalancer-module/modules/loadbalancer/main.tf#L179

Reply:
😳 you're right